sample/Web/ServiceExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Core;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Localization;
+using Web;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class ServiceExtensions
+    {
+        public const string HttpClientName = "traduora";
+        public static IServiceCollection AddTraduora(this IServiceCollection services)
+        {
+            services.AddHttpClient(HttpClientName)
+                .ConfigureHttpClient((sp, client) =>
+            {
+                var config = sp.GetRequiredService<IConfiguration>();
+                client.BaseAddress = new Uri(config["TraduoraApi:BaseUrl"]);
+            });
+
+            services.AddTransient<ITraduoraService, TraduoraService>();
+
+            services.AddSingleton<IStringLocalizer>(sp =>
+            {
+                var config = sp.GetRequiredService<IConfiguration>();
```

Comment:
When both Config and Service is in the IoC container. The need for resolving them in registration is not needed.